maintainers/maintainer-list.nix

```diff
@@ -2930,6 +2930,12 @@
     githubId = 37375448;
     name = "Buildit";
   };
+  bwc9876 = {
```

Comment:
I would need explicit consents from @Bwc9876 and @Spoonbaker to approve this part.

Reply:
I am cool with this.

Reply:
I am also cool with this.